code/modules/vore/appearance/sprite_accessories_vr.dm

```diff
@@ -1,4 +1,4 @@
-/*
+ing /*

```

Comment:
Oopsie!

Reply:
And yet it compiles, apparently. 🤔

Reply:
It should still be fixed.

Reply:
I know, just amusing that it doesn't trigger the bot.

Reply:
```suggestion
/*
```